user-guide/Standard_Apps/IDP/Advanced_configuration/Implementing the Configuration Backup script.md

```diff
@@ -0,0 +1,197 @@
+---
+uid: ConfigurationBackupScript
+---
+
+# Implementing the Configuration Backup script
+
+## Setting up the DataMiner Configuration Archive
+
+DataMiner IDP can be used to create configuration backups from [elements](xref:About_elements) in the managed inventory. However, before backups can be taken, the DataMiner Configuration Archive needs to be set up.
```

Comment:
I feel like information is missing here on how you actually set up the DataMiner Configuration Archive. Can we add this or add a link to any existing information about this?